sgt-back-end/server.js

```diff
@@ -0,0 +1,147 @@
+import express from 'express';
+import pg from 'pg';
+
+const app = express();
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    // Allow non-SSL traffic to localhost
+    rejectUnauthorized: false,
+  },
+});
+
+app.get('/api/grades', async (req, res, next) => {
+  try {
+    const sql = `
+    select *
+    from "grades"
+    `;
+    const params = [];
+    const result = await db.query(sql, params);
+    const grade = result.rows;
+    if (!grade) {
```

Comment:
If there are no grades, this will be an empty array, so this check isn't really needed.